### DISCUSSION TITLE
关于Community Toolkit当中Messenger的一些疑问

### DISCUSSION THREAD

**Discussion by zzzdajb** [BYJRK/DotNet-Discussions#226] [MVVM]
Up你好！在看完了你的课程“如何使用 CommunityToolkit.Mvvm 中的 Messenger 来进行 ViewModel 之间的通信”之后，有人评论区提到了这个
![image](https://github.com/BYJRK/DotNet-Discussions/assets/122362831/b8e0429a-dd9d-46b4-9ab6-97d135d99b9c)
但是我不是很明白，可不可以稍微详细讲解一下这个？

**Comment by BYJRK** [comment]
这个我在Messenger那期视频里面应该就提到了吧，在后半部分，包括ObservableRecipient和IRecipient接口的用法
